rest/config.go

```diff
@@ -132,6 +133,11 @@ type DbConfigMap map[string]*DbConfig
 
 type ReplConfigMap map[string]*ReplicationConfig
 
+type PersonaConfig struct {
+	Origin   string // Canonical server URL for Persona authentication
+	Register bool   // If true, server will register new user accounts
+}
+
```

Comment:
There were conflicts with the Persona changes when rebasing, I missed this one. Removed.